demo/demo.md

```diff
@@ -2,52 +2,187 @@
+<auro-toast style="display: block; margin: 0.5rem 0;" variant="success" id="successToast-noIcon" noIcon> Successfully added lap infant  </auro-toast>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+## Multiple toasts
+The auro-toast multi-toasts use case requires the use of the auro-toaster component.
+<div class="exampleWrapper">
+<!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/multipleToasts.html) -->
+<!-- The below content is automatically added from ./../../apiExamples/multipleToasts.html -->
+<auro-button onClick="showToast('#toast-default')">Show default toast</auro-button>
+<auro-button onClick="showToast('#toast-error')">Show error toast</auro-button>
+<auro-button onClick="showToast('#toast-success')">Show success toast</auro-button>
+<auro-toaster>
+    <auro-toast id="toast-default">Default toast</auro-toast>
+    <auro-toast id="toast-error" variant="error">Unable to add lap infant. Please try again</auro-toast>
+    <auro-toast id="toast-success" variant="success">Successfully added lap infant</auro-toast>
+</auro-toaster>
+<!-- AURO-GENERATED-CONTENT:END -->
+</div>
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/multipleToasts.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/multipleToasts.html -->
+
+```html
+<auro-button onClick="showToast('#toast-default')">Show default toast</auro-button>
+<auro-button onClick="showToast('#toast-error')">Show error toast</auro-button>
+<auro-button onClick="showToast('#toast-success')">Show success toast</auro-button>
+<auro-toaster>
+    <auro-toast id="toast-default">Default toast</auro-toast>
+    <auro-toast id="toast-error" variant="error">Unable to add lap infant. Please try again</auro-toast>
+    <auro-toast id="toast-success" variant="success">Successfully added lap infant</auro-toast>
+</auro-toaster>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+## Dynamic toasts
+If you choose to implement toasts dynamically, the following is an example of how to implement dynamic toasts using Vue.
+```js
+interface Toast {
+  variant: string,
+  message: string,
+  visible: boolean
+}
+
+const toasts = ref<Array<Toast>>([]);
 
+// Function that adds toasts to the toasts array
+handleToastEvent(toast) {
+  toasts.push(toast);
+}
+
+// Function that removes the toast from the DOM
+handleOnToastClose(event) {
+  // In event.details comes the ID of the auro-toast that was closed (visible was set to false)
+  const id = event.details;
+  toasts.value = toasts.filter((t) => t.id !== id)
+}
+```
+```html
+<auro-toaster>
+     <auro-toast 
+      v-for="(toast, i) in toasts"
+      :variant="toast.variant"
+      :visible="toast.visible"
+      :id="i" 
+      @onClose="handleOnToastClose">
+        {{ toast.message }}
+     </auro-toast>
+  <auro-toaster>
+```
+
 ## Recommended Use and Version Control
```

Comment:
The demo here is not working as expected.

Reply:
i pushed up changes to fix this on this commit: https://github.com/AlaskaAirlines/auro-toast/commit/c96cf10f03524f9f954ad160ef09d581e4fa806e#diff-b18ccd15ec95e2a9c4fead5bca9085042af688361a24d3463c27138e3c797a5b